src/vm-vcpu/src/vcpu/msrs.rs

```diff
@@ -37,5 +37,8 @@ pub(crate) fn create_boot_msr_entries() -> Vec<kvm_msr_entry> {
             data: u64::from(MSR_IA32_MISC_ENABLE_FAST_STRING),
             ..Default::default()
         },
-    ]
+    ];
+
+    // This unwrap is safe because we are creating valid a fixed size array with msrs entries.
+    Msrs::from_entries(&raw_msrs).unwrap()
```

Comment:
I would still propagate the error here since we can't be sure `from_entries` won't return in the future other errors as well.

Reply:
Done.